examples/federated-learning/tff/nvflare/README.md

```diff
@@ -0,0 +1,283 @@
+nvflare-helper-8f88cbf9b-tfk8c     1/1     Running   0          16h
+nvflare-server1-66c44ddb47-dhtqz   1/1     Running   0          16h
+```
+
+## 6. Submit the job
+Everything is now ready to submit and run the job. Connect to the helper to submit the job:
+
+```bash
+kubectl exec -it deploy/nvflare-helper -c nvflare-helper -- bash
+```
+
+Go to the `admin@nvidia.com` folder and connect to the infrastructure. When prompted, the username is `admin@nvidia.com`:
+
+```bash
+cd /workspace/nvfl/workspace/example_project/prod_00/admin@nvidia.com/startup
+./fl_admin.sh
+```
+
+You should be connected to the federated learning system:
+
+```bash
+User Name: admin@nvidia.com
+Trying to obtain server address
+Obtained server address: server1:8003
+Trying to login, please wait ...
+Logged into server at server1:8003 with SSID: ebc6125d-0a56-4688-9b08-355fe9e4d61a
+Type ? to list commands; type "? cmdName" to show usage of a command.
+>
+```
+
+When connected, you can list the jobs submitted to the cluster by using the `list_jobs` command. To start the TensorFlow job, just type the command `submit_job` with the name of the job you want to run, here `hello-tf2`:
+
+```bash
+> submit_job hello-tf2
+Submitted job: c8973f05-8787-41c5-8568-ecc15c7683b2
+Done [262650 usecs] 2024-05-23 09:47:04.543903
+```
+
+The job should be running now:
+
+```bash
+> list_jobs
+-----------------------------------------------------------------------------------------------------------------------------
+| JOB ID                               | NAME      | STATUS             | SUBMIT TIME                      | RUN DURATION   |
+-----------------------------------------------------------------------------------------------------------------------------
+| c8973f05-8787-41c5-8568-ecc15c7683b2 | hello-tf2 | RUNNING            | 2024-05-23T09:47:04.488652+00:00 | 0:00:11.978134 |
+-----------------------------------------------------------------------------------------------------------------------------
+Done [136046 usecs] 2024-05-23 09:47:17.630953
+```
+
+You can verify the job ran successfully with the `list_jobs` command:
+
+```bash
+> list_jobs
+-----------------------------------------------------------------------------------------------------------------------------
+| JOB ID                               | NAME      | STATUS             | SUBMIT TIME                      | RUN DURATION   |
+-----------------------------------------------------------------------------------------------------------------------------
+| c8973f05-8787-41c5-8568-ecc15c7683b2 | hello-tf2 | FINISHED:COMPLETED | 2024-05-23T09:47:04.488652+00:00 | 0:01:44.335456 |
+-----------------------------------------------------------------------------------------------------------------------------
+Done [56885 usecs] 2024-05-23 09:49:15.420097
+```
+
+Alternatively, you can also inspect the logs of both the server and the clients:
+
+```bash
```

Comment:
Can you shorten this block? It's likely not necessary to show the whole log. Focus on the parts that matter, explaining to users what they are.

Reply:
Done